lib/adyen/version.rb

```diff
@@ -1,4 +1,4 @@
 module Adyen
   NAME = "adyen-ruby-api-library"
-  VERSION = "2.0.0".freeze
+  VERSION = "2.0.1".freeze
```

Comment:
@crrood as payment methods will break with v49 we need to create a mayor release for this so 3.0.0. Let's update all the endpoints to there latest version. can you update ?